packaging/ansible-runner-service-project/project/roles/ovirt-host-upgrade/tasks/main.yml

```diff
@@ -36,6 +36,10 @@
 - name: Remove network-scripts-openvswitch 2.11 package
   shell: rpm -e --nodeps network-scripts-openvswitch2.11
   ignore_errors: true
+  when:
+    - el_ver|int >= 8
```

Comment:
I'd put there only EL8, because EL7 we don't upgrade and EL9 will already have 2.15

Reply:
done